src/components/App/__tests__/ModalApp.test.tsx

```diff
@@ -14,6 +14,7 @@ const defaultOptions: SDKOptionsWithRenderData = {
     { type: 'welcome' },
     { type: 'document' },
     { type: 'face' },
+    { type: 'data' },
```

Comment:
Are there some docs I can read around this?

Reply:
created a high priority bug to rewrite this component asap https://onfido.atlassian.net/browse/IX-2297

Reply:
No spec doc as such, Only recently this has been added.(https://www.figma.com/file/0pKZEKlY7F9TSKMr6CAXBr/Profile-Data-Capture-Screens?node-id=93%3A5186)